### DISCUSSION TITLE
OPC UA client security policy

### DISCUSSION THREAD

**Discussion by ondrej-nemec** [eclipse-milo/milo#1043] [General]
Hi,

How can I set `MessageSecurityMode.SignAndEncrypt` and `SecurityPolicy.Basic256Sha256` to OPC UA client?

Thank you

**Comment by kevinherron** [comment]
The server must be offering an endpoint with these settings, and you would just select that endpoint to connect to when creating your client.

**Reply by kevinherron** [reply]
See https://github.com/eclipse/milo/wiki/Client#connecting

**Reply by ondrej-nemec** [reply]
Thanks, now its working
